pkg/sass-parser/lib/src/expression/__snapshots__/color.test.ts.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`a color expression toJSON 1`] = `
+{
+  "inputs": [
+    {
+      "css": "@#{#00f}",
```

Comment:
Is it intended for `@` to be here?

Reply:
Yeah, because we didn't have variable statements early on the way we parse expressions is by creating an interpolation in an unknown at-rule.